src/features/favorites/pages/__tests__/useFavoritesResults.test.tsx

```diff
@@ -12,14 +13,29 @@ import { server } from 'tests/server'
 
 import { getPaginatedFavorites, useFavoritesResults } from '../useFavoritesResults'
 
+jest.mock('features/auth/AuthContext')
+const mockUseAuthContext = useAuthContext as jest.MockedFunction<typeof useAuthContext>
+
+const mockPaginatedFavoritesResponseSnap = paginatedFavoritesResponseSnap
+jest.mock('features/favorites/pages/useFavorites', () =>
+  Object.assign(jest.requireActual('../../../../features/favorites/pages/useFavorites'), {
+    useFavorites: () => ({
+      data: mockPaginatedFavoritesResponseSnap,
+    }),
+  })
+)
```

Comment:
Suggestion pour la lisibilité (il m'a fallu un petit moment pour comprendre ce que tu voulais faire avec l'Object.assign
```suggestion
jest.mock('features/favorites/pages/useFavorites', () => ({
    ...jest.requireActual('../../../../features/favorites/pages/useFavorites'),
    useFavorites: () => ({
      data: mockPaginatedFavoritesResponseSnap,
    }),
  })
)
```

Reply:
Si j'ai utilisé cette syntax c'est à cause du typing: `TS2698: Spread types may only be created from object types.`